components/Hero/Hero.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Image from "next/image";
+import { Link } from "components/Link/Link";
+import { CTA, Photo } from "types/layout-schema-types";
+import { urlFor } from "lib/client";
+
+export interface HeroSection extends React.HTMLProps<HTMLElement> {
+  cta: CTA;
+  description: string;
+  photo: Photo;
+  title: string;
+}
+
+export const Hero = ({ title, description, photo, cta }: HeroSection) => {
+  return (
+    <section className="py-10">
```

Comment:
If you just move the class `py-10` to the element in the next line (`<div className="flex ...`), you will have the same output and will avoid to have an extra wrapper.

Reply:
> If you just move the class `py-10` to the element in the next line (`<div className="flex ...`), you will have the same output and will avoid to have an extra wrapper.

I mean, I get that but it would be not good if, for example, in the future,  you change the background-color of the entire Hero. If it's not a problem, then fine. What do you think @LPRegen 
![4FABFii](https://user-images.githubusercontent.com/28108101/233792974-4ff438c0-b8b8-47d6-90b6-7355facd90d0.png)

Reply:
Hi @alexxuss1996 , IMO for the moment we should stick to the design. If in the feature there is a need to change its background, we can modify it.

@TonyMckes what do u think?

Reply:
@LPRegen, I think @alexxuss1996 is just showing you an example of how it would look with the `section` holding all the styles as you suggested (it won't cover the entire width of the screen). 

I think that either way, it doesn't matter too much as long as the content (heading, paragraph and image) is within the `seccion`.

> Btw, I think this is ready. We are just trying to figure out how the image is going to look in bigger screens.

Reply:
Oh I see now, sry!
Now makes sense to have all inside that `section`.